source/sass/blocks/advantages.scss

```
@@ -0,0 +1,62 @@
+@use "../global/var.scss";
+
+.advantages__item {
+  background-color: var.$gray-light;
+}
+
+.advantages__item--image {
+  background-color: var.$blue;
+}
```

По всем модификаторам, лучше писать вот так
 ```suggestion
.advantages__item {
  background-color: var.$gray-light;
  &--image {
    background-color: var.$blue;
  }
}


```

---

Done